_pages/dev/event-service.md

```diff
@@ -124,3 +133,60 @@ onClick() {
   this.events.dispatch(new CustomUIEvent(...));
 }
 ```
+
+## Event Type Inheritance
+
+{% capture version_note %}
+{{ site.version_note_part1a }} 3.1 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+Parent events, also called umbrella events, allow you to group similar events under one common event. The parent event can be an abstract class or a regular class. By subscribing to the parent event, you get emissions from all of the child events that inherit it.
+
+For example, `PageEvent` from `@spartacus/storefrontlib` is a parent event, and all of the child page events, such as `HomePageEvent`, `CartPageEvent`, and `ProductDetailsPageEvent`, inherit from this parent. If you subscribe to `PageEvent`, you will get emissions from all of the child page events that inherit from `PageEvent`. The following is an example:
+
+```typescript
+eventService.get(PageEvent).subscribe(...) // receives all page events
+```
+
+All events should inherit from `CxEvent` in `@spartacus/core`, as shown in the following example:
```

Comment:
..._from `CxEvent` in `@spartacus/core`_...
To me, this sounds like: all events that are created in `projects/core` lib should inherit from the `CxEvent`, while the rest of events (that are created in other libs) do not have to.

Reply:
Good point! I changed it to 

> All events should inherit from the `CxEvent` that is in `@spartacus/core`...